### ISSUE TITLE
How to add loss, optimizer, scheduler to config

### ISSUE THREAD

**Issue by vpeopleonatank** [ashleve/lightning-hydra-template#150]
How should I add loss, optimizer, scheduler to config like this template  https://github.com/Erlemar/pytorch_tempest

**Comment by ashleve** [comment]
FIrstly expand current config files with your optimizer/loss/scheduler configs.

Next I suggest to just pass them to your model in train.py:
```python
# Init Lightning model
model: LightningModule = hydra.utils.instantiate(config.model, optimizer_conf=config.optimizer, scheduler_conf=config.scheduler, loss_conf=config.loss)
```
And then you can initialize those objects in your model using hydra instantiate, for example:
```python
optimizer = hydra.utils.instantiate(optimzer_conf)
```

**Comment by vpeopleonatank** [comment]
Thanks for your examples.
